### DISCUSSION TITLE
您好，能帮我解释下这部分的含义吗？我能对应上paper这块，但就是不知道什么意思。

### DISCUSSION THREAD

**Discussion by buaacarzp** [STVIR/pysot#330] [General]
下面的代码我看不懂，为什么已经得到了修正后的bbox了，还要进行下面的操作？
![image](https://user-images.githubusercontent.com/43240561/77132978-5b741c80-6a9c-11ea-8b2f-7d240fa50946.png)
我知道有一部分对应论文的这块，但是我不理解这个含义是什么意思：

![image](https://user-images.githubusercontent.com/43240561/77133033-8d857e80-6a9c-11ea-9179-0aea182d2f12.png)
希望大佬能帮我解释下，我真的感激不尽。

**Comment by ZhiyuanChen** [comment]
我们假设追踪的是一个刚体，即其尺寸、形状在短时间内不会发生较大的变动。因此，我们对这样的改变进行惩罚。

**Comment by buaacarzp** [comment]
> 我们假设追踪的是一个刚体，即其尺寸、形状在短时间内不会发生较大的变动。因此，我们对这样的改变进行惩罚。

您好，我想问下，在tracking阶段，得到的bbox的位置不就可以了吗？为什么还要加上模板的中心点坐标位置呢？
```
cx = bbox[0] + self.center_pos[0] #-15+345 ,模板的中心点
cy = bbox[1] + self.center_pos[1]
```

**Comment by buaacarzp** [comment]
I got it.
